lib/prepare.js

```diff
@@ -87,7 +87,11 @@ async function cleanupDirectories(directories) {
 
                 // ensure files can be read by host system by running chmod before copy
                 filter: src => {
+                    if (src.startsWith(directories.postprocessing)) {
+                        return false;
+                    }
                     fse.chmodSync(src, 0o766);
+                    console.log(`WORKER_TEST_MODE: copying ${src} to /out`);
```

Comment:
Is this an information about the worker's test mode or a debug leftover? Because I don't see a `copy`in the immediate neighboring lines here.

Reply:
The `fse.copy()` is above - this is a filter callback to decide what files in the subtree to copy or not.

The log statement is meant to stay in - to track future issues like this.